Project.toml

```diff
@@ -68,6 +68,7 @@ Logging = "1.10"
 Makie = "0.20"
 Markdown = "1.10"
 ModelingToolkit = "8.75, 9"
+ModelingToolkitStandardLibrary = "2.7"
```

Comment:
This should be gated int Downstream

Reply:
I've added bounds to `test/downstream/Project.toml` in https://github.com/SciML/SciMLBase.jl/pull/689/commits/940ea7820b6108c745bd30aee51a1cd1eb1d7461, should I remove anything from the regular test environment or do i need to declare these in both places?

Reply:
https://github.com/SciML/SciMLBase.jl/pull/689/commits/d061ce47f10d99f78850e08b820727c6228b3c8b does that